### ISSUE TITLE
This is persistent?

### ISSUE THREAD

**Issue by chaoticvibe** [emcniece/cordova-plugin-indexedDB2#1]
This is persistent db, that dont will be cleaned if users close the app?
I need something very persistent, it's looks great, congratulations.
--- It's got storage usage limits?
--- You can give me advice? With your experience with this plugin, it's a good choice to simple tasks (localStorage will use many memory for my scenario)?
I appreciate your time and attention, thank you

**Comment by emcniece** [comment]
Yes, this is persistent storage. Its limits will be whatever IndexedDB uses in browser; according to [Browser storage limits and eviction criteria](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API/Browser_storage_limits_and_eviction_criteria) it might be 50% of the total hard drive space. There might be some differences between iOS and Android native browsers.

IndexedDB is a suitable alternative for localStorage 👍
